datasafe-rest-impl/src/main/java/de/adorsys/datasafe/rest/impl/controller/InboxController.java

```diff
@@ -92,6 +120,11 @@ public void deleteFromInbox(@RequestHeader String user,
      * list files in users' INBOX.
      */
     @GetMapping(value = "/{path:.*}", produces = APPLICATION_JSON_VALUE)
+    @ApiOperation("List files in inbox")
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "List command successfully completed"),
+            @ApiResponse(code = 403, message = "Access denied")
```

Comment:
403 has default message, not necessary to add it

Reply:
removed